packages/admin-ui/src/pages/system/components/Notifications/EthicalMetrics.tsx

```diff
@@ -61,18 +105,26 @@ export default function EthicalMetrics() {
       });
       setReqStatusDisable({ result: true });
       ethicalMetricsConfig.revalidate();
+      setEthicalMetricsOn(false);
     } catch (e) {
       setReqStatusDisable({ error: e });
       console.error("Error on registerEthicalMetrics", e);
     }
   }
 
+  function enableEthicalSwitch() {
```

Comment:
I dont get this, its useless to have a function that just calls another function. Remove it if possible